e2e/app.spec.ts

```diff
@@ -142,5 +147,22 @@ describe('App', function() {
     // Ensure that more actions have been loaded on the page.
     actionListItemsFinder = await actionsListPage.getActionListItems();
     expect(actionListItemsFinder).to.have.lengthOf(6);
+
+    // Click on the first action item
+    actionListItemsFinder[0].click();
```

Comment:
My understanding of the page object pattern is that we should avoid doing this kind of thing (clicking, interacting with the DOM) in the test itself, but rather add a `goToAction(n)` method in the `ActionsListPageObject ` class to do it for us, which (hopefully) decouples the test itself from the DOM (at least a little more).

This `goToAction(n)` method would retrieve the list, click on the n-th action and directly return the `ActionPageObject` representing the new page, basically this:

```
async goToAction(n) {

  const actions = await actionsListPage.getActionListItems();
  actionListItemsFinder[0].click();

  const actionPage = new ActionPageObject('action-page');
  await presenceOf(actionPage.getPage());

  return actionPage;
}
```

Note that I have no problem with what the test does, it's just a pattern/style concern.

Reply:
Okay. I tried implementing something like this in the latest commits. Tell me if this is what you had in mind.
Maybe we ought to extend this pattern to other interactions done in the test ? (like clicking on the map markers or the popover backdrop) ?

Reply:
> Okay. I tried implementing something like this in the latest commits. Tell me if this is what you had in mind.
> Maybe we ought to extend this pattern to other interactions done in the test ? (like clicking on the map markers or the popover backdrop) ?

Well, you responded to that in your comments on the next PR. I will try something on that one, then.